### PR TITLE
quantization doc

### PR DESCRIPTION
**Pull request by chenfucn**
Augment quantization documents, explaining quantization pre-processing and loss debugging

### CODE REVIEW

**Comment by natke** [comment]
Thanks @chenfucn and @yufenglee! Would you mind pushing your branch to the gh-pages branch in your fork, and this will stage the document as an HTML page

**Comment by chenfucn** [comment]
> Thanks @chenfucn and @yufenglee! Would you mind pushing your branch to the gh-pages branch in your fork, and this will stage the document as an HTML page

@natke Do you mean I need to abandon this PR, move all the changes to the gh-pages branch in another fork, and start a new PR from that fork?

**Comment by natke** [comment]
> @natke Do you mean I need to abandon this PR, move all the changes to the gh-pages branch in another fork, and start a new PR from that fork?

No, not at all. If you have an onnxruntime fork, all you need to is push to that fork ie git push origin cfu_qdqdoc:gh-pages

But if you don't have a fork, don't worry about it!

**Comment by chenfucn** [comment]
https://chenfucn.github.io/onnxruntime/

File: docs/performance/quantization.md [hunk 1]
```diff
@@ -48,18 +48,112 @@ The picture below shows the equivalent representation with the QOperator and QDQ
...
+debugging.
 
-* **Static quantization**: leverages calibration data to calculate the quantization parameters of activations. Our quantization tool supports three calibration methods: MinMax, Entropy and Percentile. Please refer to [`calibrate.py`](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/calibrate.py) for details.
+### Pre-processing
+{: .no_toc}
+
+Pre-processing is to transform a float32 model to prepare it for quantization. It consists
+of the following three optional steps:
+
+1. Symbolic shape inference. This is best suited for transformer models.
```
Comment (natke) [inline docs/performance/quantization.md:62]: Should these run in order? The numbered list makes it seem so.  If not, maybe change to a bulleted list

**Review by natke — APPROVED** [review]

**Comment by chenfucn** [comment]
I don't have the access rights for merging this. Who should I ask?

**Comment by natke** [comment]
> I don't have the access rights for merging this. Who should I ask?

I can merge it

**Comment by chenfucn** [comment]
Thank you @natke!

**Pull request merged**